apollo-compiler/src/test/graphql/com/example/__schema/java/operationBased/__schema/type/ColorInput.java.expected

```diff
@@ -114,44 +114,48 @@ public class ColorInput {
      * Red color
      */
     public Builder red(@Nullable Integer red) {
-      this.red = Optional.presentIfNotNull(red);
+      this.red = red;
       return this;
     }
 
     /**
      * Green color
      */
     public Builder green(@Nullable Double green) {
-      this.green = Optional.presentIfNotNull(green);
+      this.green = green;
       return this;
     }
 
     /**
      * Blue color
      */
     public Builder blue(@Nullable Double blue) {
-      this.blue = Optional.presentIfNotNull(blue);
+      this.blue = blue;
       return this;
     }
 
     /**
      * for test purpose only
      */
     public Builder enumWithDefaultValue(@Nullable Episode enumWithDefaultValue) {
-      this.enumWithDefaultValue = Optional.presentIfNotNull(enumWithDefaultValue);
+      this.enumWithDefaultValue = enumWithDefaultValue;
       return this;
     }
 
     /**
      * Circle ref to review input
      */
     public Builder reviewRefInput(@Nullable ReviewRefInput reviewRefInput) {
-      this.reviewRefInput = Optional.presentIfNotNull(reviewRefInput);
+      this.reviewRefInput = reviewRefInput;
       return this;
     }
 
     public ColorInput build() {
-      return new ColorInput(red, green, blue, enumWithDefaultValue, reviewRefInput);
+      return new ColorInput(Optional.presentIfNotNull(red),
```

Comment:
(Not really related to nullable annotations!). This changes how builders work. I think this way is a bit nicer because previously you would be required to call every method in the builder, otherwise it could lead to NPEs. 
With this change, not calling a method is equivalent to it being absent. WDYT?

Reply:
LGTM! Ping @dwbutler since I believe you're one of the first user of the Java operation builders. Does that work for you?

Reply:
Actually there's an issue when wanting to pass `null`. Will amend!

Reply:
That behavior makes sense to me!